hs_core/hydroshare/utils.py

```diff
@@ -422,18 +422,24 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id, set_to_private=False):
     # This makes the assumption that the destination is in the same exact zone.
     # Also, bags and similar attached files are not copied.
     istorage = src_res.get_irods_storage()
-    src_coll = os.path.join(src_res.root_path, 'data')
-    dest_coll = os.path.join(tgt_res.root_path, 'data')
-    istorage.copyFiles(src_coll, dest_coll)
 
+    # This makes an exact copy of all physical files.
+    src_files = os.path.join(src_res.root_path, 'data')
+    # This has to be one segment short of the source because it is a target directory.
+    dest_files = tgt_res.root_path
+    istorage.copyFiles(src_files, dest_files)
+
+    src_coll = src_res.root_path
+    tgt_coll = tgt_res.root_path
```

Comment:
@alvacouch I think it is better to combine ```dest_files``` and ```tgt_coll``` into one variable since they have the same value. How about combining them into one variable called ```dest_coll```?